File: app/src/main/java/com/aarushi/restapiretrofitjson/Models/Comment.java
package com.aarushi.restapiretrofitjson.Models;

/**
 * Created by hp on 7/2/2017.
 */

public class Comment {
    String body;
    String email;
    String name;

    public Comment(String body, String email, String name) {
        this.body = body;
        this.email = email;
        this.name = name;
    }

    public String getBody() {
        return body;
    }

    public String getEmail() {
        return email;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/codingblocks/restapiretrofitjson/models/Comment.kt
package com.codingblocks.restapiretrofitjson.models

/**
 * Created by Suraj on 6/29/2017.
 */

data class Comment (
        val postId: Int,
        val id: Int,
        val name: String,
        val email: String,
        val body: String
)